docs/sources/setup-grafana/configure-security/configure-authentication/github/index.md

```diff
@@ -64,6 +64,14 @@ automatically signed up.
 
 You can also use [variable expansion]({{< relref "../../../configure-grafana/#variable-expansion" >}}) to reference environment variables and local files in your GitHub auth configuration.
 
+### Access token expiration check and configure refresh token
```

Comment:
This section should be titled something else because there is no check and there is no configure refresh token task. Maybe "Github access tokens" - or something like that?

Reply:
I changed to `GitHub refresh token`, because this section is about the refresh token, what do you think?